lib/core/registry.ex

```diff
@@ -81,6 +81,55 @@ defmodule TelemetryMetricsPrometheus.Core.Registry do
 
   defp validate_consistent_units(_units, _), do: :ok
 
+  def validate_distribution_buckets!(%Metrics.Distribution{} = metric) do
+    reporter_options = metric.reporter_options
+
+    unless reporter_options != nil do
+      raise ArgumentError, "expected reporter_options to be on metric"
```

Comment:
Let's try to provide more info for the user on this error and the buckets one so that the user knows what the error applies to. Maybe something like `reporter_options and the buckets property are required to be defined for Distribution metrics`. Whatever we go with, we should make it clear it applies to the Distribution metric.

Reply:
Sure thing. Addressed on the latest commit. Error message is: `reporter_options with a valid buckets property are required to be defined for Distribution metrics`. Let me know if it needs a change.